### PR TITLE
Frontier - Add JoeDAO to Frontier

### PR DESCRIPTION
**Pull request by joedaoofficial**

### CODE REVIEW

**Comment by joedaoofficial** [comment]
@JeremyParish69 I believe I followed the instructions correctly now :) please let me know if that is the case.

**Comment by JeremyParish69** [comment]
> @JeremyParish69 I believe I followed the instructions correctly now :) please let me know if that is the case.

Yes, now ASVT is there and you can add JOE after line 1252

**Comment by joedaoofficial** [comment]
> 1252

Hey @JeremyParish69 do you know where I get the ics20ContractAddress? I have the cw20contractaddress.

**Comment by joedaoofficial** [comment]
> > 1252
> 
> Hey @JeremyParish69 do you know where I get the ics20ContractAddress? I have the cw20contractaddress.

okay nvmd I found out its just a standard for juno contracts to transfer assets to osmosis. Okay fixed. Think we're ready to go now @JeremyParish69 thanks for all the help. Let me know if this is complete :)

File: src/config.ts [hunk 1]
```diff
@@ -1250,6 +1250,13 @@ export const IBCAssetInfos: {
 		coinMinimalDenom: 'cw20:juno17wzaxtfdw5em7lc94yed4ylgjme63eh73lm3lutp2rhcxttyvpwsypjm4w',
 		ics20ContractAddress: 'juno1v4887y83d6g28puzvt8cl0f3cdhd3y6y9mpysnsp3k8krdm7l6jqgm0rkn',
 	},
+	{
+		counterpartyChainId: 'juno-1',
+		sourceChannelId: 'channel-169',
+		destChannelId: 'channel-47',
+		coinMinimalDenom: 'cw20:cw20:juno1n7n7d5088qlzlj37e9mgmkhx6dfgtvt02hqxq66lcap4dxnzdhwqfmgng3',
```
Comment (JeremyParish69) [inline src/config.ts:1257]: you have 'cw20:' here twice

Comment (joedaoofficial) [reply src/config.ts:1257]: ah lol

Comment (joedaoofficial) [reply src/config.ts]: > you have 'cw20:' here twice

fixed

**Comment by joedaoofficial** [comment]
fixed

**Comment by joedaoofficial** [comment]
do me a favor @JeremyParish69 if you believe we are fixed and ready to go can you let me know we are okay to set up the osmosis frontier pool?

**Comment by JeremyParish69** [comment]
> do me a favor @JeremyParish69 if you believe we are fixed and ready to go can you let me know we are okay to set up the osmosis frontier pool?

Yep, looks ready. You can use the Vercel generated preview link to create the pool. Then we can add it's pricing and add it to the Trade router before merging.

**Comment by joedaoofficial** [comment]
> > do me a favor @JeremyParish69 if you believe we are fixed and ready to go can you let me know we are okay to set up the osmosis frontier pool?
> 
> Yep, looks ready. You can use the Vercel generated preview link to create the pool. Then we can add it's pricing and add it to the Trade router before merging.

Hey @JeremyParish69 we just provided liquidity! :) Pool #718, please let me know what the next steps are

**Comment by jonator** [comment]
I noticed on the png there's an extra ring around the black circle w/ border. Is that intentional?

**Comment by joedaoofficial** [comment]
> I noticed on the png there's an extra ring around the black circle w/ border. Is that intentional?

yes

**Comment by joedaoofficial** [comment]
@JeremyParish69 are we ready to be merged?

**Comment by joedaoofficial** [comment]
Hi @JeremyParish69 following up

**Comment by JeremyParish69** [comment]
Pausing additions to Frontier until post-refactor implementation of Frontier. Then will create a new PR to update there.

**Comment by joedaoofficial** [comment]
ok

**Pull request merged**